scipy/optimize/tests/test_optimize.py

```diff
@@ -1190,6 +1190,18 @@ def test_brute(self):
         assert_allclose(resbrute[1], self.func(self.solution, *self.params),
                         atol=1e-3)
 
+    def test_1D(self):
+        # test that for a 1D problem the test function is passed an array,
+        # not a scalar.
+        def f(x):
+            # we would like the parame
```

Comment:
Is this comment incomplete? I'm guessing the purpose of the comment is to clarify that `x` should be a 1D array? The comment in the parent function might suffice in any case.